Source/Scene/ModelExperimental/ModelExperimentalNode.js

```diff
@@ -67,6 +99,46 @@ export default function ModelExperimentalNode(options) {
   initialize(this);
 }
 
+Object.defineProperties(ModelExperimentalNode.prototype, {
+  transform: {
+    get: function () {
+      return this.transform;
+    },
+    set: function (value) {
+      // TODO: Set dirty flag.
```

Comment:
At the moment, to apply the change to the transform, we'd need to call `resetDrawCommands` on the `ModelExperimental`. This is not feasible because an externally driven animation source will cause draw commands to be built per-tick.

I'm thinking this is the right time to create some infrastructure for `UpdateStages`, similar to the `PipelineStages`, although I am not exactly sure if those are best applied by the `ModelExperimental` or the `ModelExperimentalSceneGraph`.

Reply:
Does the draw command need to be updated when the matrix changes? Or could the scene graph code update the uniform map's model matrix in place?

We're going to need some draw command update infrastructure eventually but it would be good to know all the use cases before starting on it. `backFaceCulling` is one example since it affects render state and not the uniform map. I'm sure there are others.